src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlJson.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClient
+{
+    internal class SqlJson
```

Comment:
Can you add TODO comments or something that sets the expectation of what's going to come in this type?

Reply:
File removed. To be added later as part of SqlJson implementation.